e2e/cypress/integration/system_console/site_configuration/link_customization_spec.js

```diff
@@ -17,17 +18,15 @@ describe('SupportSettings', () => {
     const aboutLink = 'http://www.mattermost.org/features/';
     const helpLink = 'https://github.com/mattermost/platform/blob/master/doc/help/README.md';
     const problemLink = 'https://forum.mattermost.org/c/general/trouble-shoot';
-    const email = 'bot@mattermost.com';
 
     const defaultTosLink = 'https://about.mattermost.com/default-terms/';
-    const defaultPrivacyLink = 'https://about.mattermost.com/default-privacy-policy/';
 
     let testTeam;
 
     beforeEach(() => {
-        // # as many of the tests logout the user, ensure it's logged
-        // in as an admin before each test
+        // # Login as admin and reset config
```

Comment:
```suggestion
        cy.shouldNotRunOnCloudEdition();

        // # Login as admin and reset config
```

Reply:
Thanks and good catch! Actually, I've already moved the tests that were not for cloud to `link_customization_not_cloud_spec`. I just missed to remove the corresponding `@not_cloud` group tag here. Now fixed.